app/views/fields/has_many/_index.html.erb

```diff
@@ -16,4 +16,4 @@ as a count of how many objects are associated through the relationship.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Field/HasMany
 %>
 
-<%= pluralize(field.data.size, field.attribute.to_s.humanize.downcase.singularize) %>
+<%= pluralize(field.data.size, t("activerecord.models.#{field.attribute.to_s.singularize}", default: field.attribute.to_s.humanize.downcase.singularize)) %>
```

Comment:
One detail: it's possible to provide plurals for translations like this one. In this case, it would look like this:

```yaml
es:
  activerecord:
    models:
      order:
        one: Pedido
        other: Pedidos
```

In this case, your code would fail with `NoMethodError undefined method `pluralize' for {:one=>"Patronisation", :other=>"Patronisations"}:Hash`. To avoid this I think we can just add a `count`, like follows:

```suggestion
<%= pluralize(field.data.size, t("activerecord.models.#{field.attribute.to_s.singularize}", default: field.attribute.to_s.humanize.downcase.singularize, count: field.data.size)) %>
```

What do you think?

Reply:
Oh! I know what do you mean, that's the one thing I haven't considered because Chinese doesn't have singular and plural problems XD.  It looks good to me, thanks.